### ISSUE TITLE
URIs for metadata profiles

### ISSUE THREAD

**Issue by cessda-bitbucket-importer** [cessda/cessda.cvs.two#267]
**Original report on BitBucket by John Shepherdson (GitHub: [john-shepherdson](https://github.com/john-shepherdson)).**

----------------------------------------

**From:**Taina Jääskeläinen \(TAU\)   
**Sent:**11 November 2020 12:30  
**To:**Shepherdson, John W   
**Subject:**Controlled vocabulary URLs 

Hi,

the CMM User Guide and the CDC profiles are to be published by early December.

For controlled vocabularies elements, it is mandatory to have an URI in the metadata. It needs to be either a resolvable URN or a URL used in the API.

 

So basically, I would need information asap on what URNs or URLs are to be entered into the metadata before  the MDO can publish the CMM user guide and the CDC profiles.

As far as I can see, they cannot be DDI Alliance URLs.

 

There is no CMM planned next year, that’s why we need to have the URL information right this time.

 

Ad hoc, could not figure out find this information in the swagger.

 

All the best,

Taina
